micro-frontends/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "openmrs-mfe-host",
+  "version": "0.0.1",
+  "description": "Hosting microfrontends for openmrs",
+  "main": "dist/index.js",
+  "author": "bahmni@thoughtworks.com",
+  "license": "AGPL-3.0",
+  "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "@babel/plugin-transform-runtime": "^7.22.4",
+    "@babel/preset-env": "^7.22.4",
+    "@babel/preset-react": "^7.22.3",
+    "@testing-library/react": "12.1.5",
+    "babel-loader": "^9.1.2",
+    "mini-css-extract-plugin": "^2.7.6",
+    "resolve-url-loader": "^3.1.0",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4",
+    "css-loader": "5.2.6",
+    "sass": "^1.63.4",
+    "sass-loader": "10.1.0"
+  },
+  "dependencies": {
+    "angular-component": "^0.1.3",
+    "react2angular": "^4.0.6",
+    "bahmni-carbon-ui": "0.1.0",
+    "carbon-components": "^10.19.0",
+    "carbon-components-react": "^7.25.0",
+    "react": "16.14.0",
+    "react-dom": "16.14.0"
+  },
+  "peerDependencies": {
+    "react-dom": "16.14.0",
+    "react2angular": "^4.0.6"
+  },
+  "scripts": {
+    "build": "webpack --mode production"
```

Comment:
Can we also include a test coverage, test and lint script in the package.json file?